lib/main_common.pm

```diff
@@ -585,6 +588,8 @@ sub load_jeos_tests {
     loadtest 'qa_automation/patch_and_reboot' if is_updates_tests;
     replace_opensuse_repos_tests if is_repo_replacement_required;
     loadtest 'console/verify_efi_mok' if get_var 'CHECK_MOK_IMPORT';
+    # zypper_ref needs to run on jeos-containers. the is_sle is required otherwise is scheduled twice on o3
+    loadtest "console/zypper_ref" if (get_var('CONTAINER_RUNTIME') && is_sle);
```

Comment:
Why checking `CONTAINER_RUNTIME`?

Reply:
This test was scheduled before in JeOS container tests using yaml. In O3 it was scheduled in some other way (therefore the `is_sle` condition), that's why this is kept like this. I'd like to wait for @mloviska to see if we can skip this test at all...

Reply:
I believe it should not be included here as it comes either from `replace_opensuse_repos_tests` (o3) or `load_consoletests`